wfl/autoparallelize/autoparainfo.py

```diff
@@ -57,9 +57,14 @@ def update_defaults(self, def_kwargs):
         for k in AutoparaInfo._kwargs:
             if not hasattr(self, k):
                 setattr(self, k, def_kwargs.pop(k, AutoparaInfo._kwargs[k]))
+            elif k in AutoparaInfo._kwargs:
```

Comment:
Isn't this if unnecessary? It's inside a loop of `k in AutoparaInfo._kwargs`?

Reply:
that's right, it should just be else in that case. Will update.

Reply:
OK, now I understand.  Yes - if we change the `elif` to `else`, I think it'll be good to go.